pkg/sql/queue_offsets_adapter_postgresql.go

```diff
@@ -0,0 +1,63 @@
+package sql
+
+import (
+	"fmt"
+
+	"github.com/lib/pq"
+)
+
+// PostgreSQLQueueOffsetsAdapter is an OffsetsAdapter for the PostgreSQLQueueSchema.
+type PostgreSQLQueueOffsetsAdapter struct {
```

Comment:
I guess it would be good to add it to watermill.io - leaving to not forget

Reply:
I plan to, but wanted to merge this one first so the docs src-link works.